source/_components/sensor.history_values.markdown

```diff
@@ -0,0 +1,141 @@
+
+# {% linkable_title Configuration %}
+{% raw %}
+```yaml
+# Example configuration.yaml entry
+sensor:
+- platform: history_values
+  name: "Peak Energy Usage Today"
+  type: 'peak'
+  target_entity_id: 'sensor.power_consumption'
+  start: >
+    {% set today %}
+      {{- now().year }}-{{ now().month }}-{{ now().day -}}
+    {% endset %}
+    {{ as_timestamp(strptime(today, '%Y-%m-%d')) }}
+  end: "{{ as_timestamp(now()) }}"
+  update:
+    minutes: '/5'
+    seconds: 0
+```
+{% endraw %}
+{% configuration %}
+name:
+  description: "Name of the sensor."
+  required: true
+  type: string
+type:
+  description: "Type of the sensor. Possible values are `average`, `low`, `peak` or `range`."
+  required: true
+  type: string
+target_entity_id:
+  description: "The entity for which values will be looked up in the history."
+  required: true
+  type: string
+unit_of_measurement:
+  description: "The unit of measurement to use for the sensor."
+  required: false
+  type: string
+  default: "Unit of measurement from the target entity."
+start:
+  description: "A template which returns a timestamp specifying the start of the measurement period. Exactly 2 of `start`, `end` and `duration` must be specified."
+  required: false
+  type: template
+end:
+  description: "A template which returns a timestamp specifying the end of the measurement period. Exactly 2 of `start`, `end` and `duration` must be specified."
+  required: false
+  type: template
+duration:
+  description: "A time period indicating the duration of the measurement. Exactly 2 of `start`, `end` and `duration` must be specified."
+  required: false
+  type: time
+update:
+  description: "The update schedule of the `history_values` sensor. Must have at least one key if specified."
+  required: false
+  type: map
+  keys:
+    at:
+      description: "The time of day on which the sensor will be updated."
+      required: exclusive
+      type: time
+    hours:
+      description: "The hours to match for an update of the sensor."
+      required: false
+      type: [integer, string]
+    minutes:
+      description: "The minutes to match for an update of the sensor."
+      required: false
+      type: [integer, string]
+    seconds:
+      description: "The seconds to match for an update of the sensor."
+      required: false
+      type: [integer, string]
+{% endconfiguration %}
+
+## {% linkable_title Sensor Types %}
+The following types are supported:
+* `average` will calculate the time-weighted average of the sensor in the target time period. Periods of time when the state of the target entity is `unknown` are disregarded.
+* `low` will show the lowest value of the sensor in the target time period.
+* `peak` will show the highest value of the sensor in the target time period.
+* `range` will show the range of the sensor in the target time period. This is the same as the value of `peak` minus the value of `low`.
```

Comment:
range is not the appropriate word (can't remember the correct term in english) 

range would be 2 numbers

Reply:
I'm open to suggestions if anyone can think of a better name for it...

Reply:
amplitude is the right word!

Reply:
Actually, now I'm not so sure anymore. Merriam-Webster defines amplitude as 'the extent or range of a quality, property, process, or phenomenon' suggesting they are synonyms. Maybe a native speaker could shine some light on this?

Reply:
"In meteorology, the range or difference between the maximum and minimum values of the temperature, pressure, or other meteorological element within a definite time, such as a day, a month, or a year."

Reply:
Reads like they are synonyms to me...